src/Components/DrillListPage.js

```diff
@@ -14,10 +14,22 @@ const secondsToMinutes = seconds => {
   return date.getMinutes();
 };
 
+const filtered_drills = data => {
+  const drills = data.drills;
+  const filter = data.filter;
+  if ('type' in filter) {
+    return drills.filter(drill => drill.type === filter.type);
+  }
+  if ('training' in filter) {
```

Comment:
I think that i you extract a `TrainingListPage` component, this block becomes useless

Reply:
I do have a TrainingListPage component to display training objects.
But then when I select one of these training from the list, I want to display the drills of the training on something that looks like a DrillListPage, except the filtering is different (I want to display only drills in the training). That's why I think I needed this block.
Should I do a dedicated trainingPage instead ?

Reply:
Then maybe we should extract a `DrillList` component that is responsible for displaying a list of drills passed as props?

And leave it to `DrillListPage` and `TrainingPage` to filter which drills are to be displayed and use this `DrillList` ?